CONTRIBUTORS

```diff
@@ -23,3 +23,4 @@ tsukumi
 bbepis
 Pccode66
 Ashish
+hheimbuerger
```

Comment:
remove this. I will add it once your code is merged. Keeping this in the PR makes it necessary to conflict resolve frequently